scss/application.scss

```diff
@@ -1,3 +1,5 @@
 @import 'equalizer';
 @import 'styles';
 @import 'titles';
+@import 'colors';
```

Comment:
place colors as the second import, this allows it to be used by all the other files, the equalizer shouldn't care about the application colors but general styles, titles and techs should.